addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -24,7 +24,7 @@ extends Node
 export var enabled := true setget set_enabled, get_enabled
 
 ## Player radius
-export var player_radius := 0.4
+export var player_radius := 0.4 setget set_player_radius, get_player_radius
```

Comment:
You can leave out the `get_player_radius` seeing it just gets the player radius, you can have property definitions with just a setter and Godot will use the default getter.

Reply:
So omitting one of the functions gives it 'default' behavior. I wish the effects of omitting setters and getters was clearer in the manual. I didn't know if omitting the setter meant default behavior or made the variable read-only, and if omitting the getter meant default behavior or making the variable write-only.

Reply:
I've removed the unnecessary getters.